pkg/tarutil/tar_test.go

```diff
@@ -48,6 +48,38 @@ func TestExtractDir(t *testing.T) {
 	}
 }
 
+func TestExtractDirNotExist(t *testing.T) {
```

Comment:
There looks to be a fair bit of duplication with other tests. Perhaps deduplicate it? Or file a bug because it's pre-existing?

Reply:
Hm I tried to separate out the common chunks into a separate function, is that what you meant?

Reply:
Looks good!